rpc/wrtc_call_queue_mongodb.go

```diff
@@ -300,16 +301,21 @@ type mongodbNewCallEventHandler struct {
 	receiveOnce sync.Once
 }
 
-func (newCall *mongodbNewCallEventHandler) Send(event mongodbCallEvent) bool {
+func (newCall *mongodbNewCallEventHandler) Send(event mongodbCallEvent, logger *zap.SugaredLogger) bool {
 	var sent bool
 	newCall.receiveOnce.Do(func() {
 		// should always work
 		select {
 		case newCall.eventChan <- event:
 			sent = true
 		default:
+			logger.Infow("Hit default select in send",
+				"Event", event.Call)
 		}
 	})
+	logger.Infow("returning from send",
+		"Event", event.Call,
+		"Sent", sent)
```

Comment:
Yeah these seem good; do you care that this might flood logs? I imagine this two info logs will be hit a lot.

Reply:
I imagine these are temporary, I'm hoping in the next week or sooner I can back them out, so it might temporarily but I think that's fine.